scripts/arc/documentation/arcdocs.md

```diff
@@ -0,0 +1,270 @@
+
+##### Option 4 - Create Azure Monitor for containers extension with advanced configuration
+
+The default requests and  limits should work for majority of the scenarios. But in case you want to tweak the resource requests and limits because your are running on clusters with a small footprint like IOT edge based clusters, you can use the advanced configurations settings to do so.
+
+Run the following command to create extension with advanced configuration:
+
+```console
+az k8s-extension create --cluster-name <cluster-name> --resource-group <resource-group> --cluster-type connectedClusters --extension-type Microsoft.AzureMonitor.Containers --name azuremonitor-containers --configuration-settings  omsagent.resources.daemonset.limits.cpu=150m omsagent.resources.daemonset.limits.memory=600m omsagent.resources.deployment.limits.cpu=1 omsagent.resources.deployment.limits.memory=750m
+```
+
+Please refer to [resource requests and limits section of Helm chart](https://github.com/helm/charts/blob/master/incubator/azuremonitor-containers/values.yaml#L87) for the supported configuration settings related to resource constraints.
+
+##### Option 5 - Create Azure Monitor for containers extension on Azure Stack Edge
+
+Kubernetes clusters on [Azure Stack Edge](https://azure.microsoft.com/products/azure-stack/edge/#benefits) require custom mount path `/home/data/docker` for container logs.
+
+Run the following command to create extension with configuration setting to indicate a custom mount path:
+
+```bash
+az k8s-extension create --cluster-name <cluster-name> --resource-group <resource-group> --cluster-type connectedClusters --extension-type Microsoft.AzureMonitor.Containers --name azuremonitor-containers --configuration-settings  omsagent.logsettings.custommountpath=/home/data/docker
+```
+#### Using Azure Resource Manager (ARM) Template
+ 
+```bash
+# 1. download arm template and parameter file
+curl -L https://aka.ms/arc-k8s-azmon-extension-arm-template -o arc-k8s-azmon-extension-arm-template.json
+curl -L https://aka.ms/arc-k8s-azmon-extension-arm-template-params -o  arc-k8s-azmon-extension-arm-template-params.json
+# 2. update parameter values in arc-k8s-azmon-extension-arm-template-params.json file.For Azure Public cloud, value of workspaceDomain is `opinsights.azure.com`
+# 3. deploy the arm template to create Azure Monitor for containers extension 
+az login
+az account set --subscription "Subscription Name"
+az deployment group create --resource-group <clusterResourceGroup> --template-file ./arc-k8s-azmon-extension-arm-template.json --parameters @./arc-k8s-azmon-extension-arm-template-params.json
+```
+
+### **Delete existing Azure Monitor for containers extension**
+
+[Deletion of the extension instance](./k8s-extensions.md#delete-extension-instance) only deletes the extension, but doesn't delete the Log Analytics workspace. The data intact within the Log Analytics resource is left intact.
+
+## Configure proxy endpoint
```

Comment:
This section not applicable since the proxy being set as part of the connected cluster creation.